int16 m1_velocity
int16 m2_velocity
int16 m3_velocity
int16 m1_current
int16 m2_current
int16 m3_current
int16 x_acelleration
int16 y_acelleration
int16 angular_velocity
int16 compass
int8 m1_dutycycle
int8 m2_dutycycle
int8 m3_dutycycle
int16 m1_setpoint
int16 m2_setpoint
int16 m3_setpoint
float64 m1_inc_control_signal
float64 m2_inc_control_signal
float64 m3_inc_control_signal
float64 m1_control_signal
float64 m2_control_signal
float64 m3_control_signal
float64 V
float64 Vn
float64 W
int16 delta_time
float64 X_robot
float64 Y_robot
float64 Theta_robot
float64 Vref
float64 Vnref
float64 Wref
